.github/workflows/tests_amd64.yaml

```diff
@@ -0,0 +1,19 @@
+name: Tests AMD64
+permissions: read-all
+on: [push, pull_request]
+jobs:
+  test-linux-amd64:
+    uses: ./.github/workflows/tests-template.yml
+
+  coverage:
+    needs: ["test-linux-amd64"]
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
Why do we need this matrix for `os` when the `runs-on` can just be specified?  It seems like it adds unnecessary complexity when there is only one entry in the matrix?

Reply:
yes, we can also do the same simplification for the coverage for windows.

Reply:
on it

Reply:
updated both 👍🏽